src/core/core.cpp

```diff
@@ -713,19 +713,54 @@ std::string wf::compositor_core_impl_t::get_xwayland_display()
 }
 
 void wf::compositor_core_impl_t::move_view_to_output(wayfire_view v,
-    wf::output_t *new_output)
+    wf::output_t *new_output, bool reconfigure)
 {
+    auto old_output = v->get_output();
     wf::view_move_to_output_signal data;
     data.view = v;
-    data.old_output = v->get_output();
+    data.old_output = old_output;
     data.new_output = new_output;
     this->emit_signal("view-move-to-output", &data);
 
+    uint32_t edges;
+    bool fullscreen;
+    wf::geometry_t vg, og, ng;
```

Comment:
since we have a ton of `*g`s here, I think it would make more sense to explicitly write what each one is. `view_g`, `old_output_g`, `new_output_g`.